rerun_py/src/catalog/catalog_client.rs

```diff
@@ -12,7 +12,7 @@ use crate::catalog::{
 };
 
 /// Client for a remote Rerun catalog server.
-#[pyclass(name = "CatalogClientInternal")]
+#[pyclass(name = "CatalogClientInternal")] // NOLINT
```

Comment:
I wish the `// NOLINT` comment had a brief description of what lint it is noping out of, and _why_